src/theme/TOC/styles.module.css

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Can you add a notice saying the package where this file is coming from?
It might become handy if we have style problems when updating.

Reply:
Sure :)